openapi.yaml

```diff
@@ -1786,6 +1839,215 @@ components:
+          "beacon":
+            {
+              "network": "preview",
+              "epoch": 123,
+              "immutable_file_number": 2345
+            },
+          "certificate_hash": "f6c01b373bafc4e039844071d5da3ace4a9c0745b9e9560e3e2af01823e9abfb",
+          "total_db_size_uncompressed": 800796318,
+          "compression_algorithm": "zstandard",
+          "cardano_node_version": "0.0.1",
+          "created_at": "2023-01-19T13:43:05.618857482Z"
+        }
+
+    CardanoDatabaseSnapshotMessage:
+      description: CardanoDatabaseSnapshot represents a Cardano database snapshot and its metadata
+      type: object
+      additionalProperties: false
+      required:
+        - merkle_root
+        - beacon
+        - certificate_hash
+        - total_db_size_uncompressed
+        - created_at
+        - locations
+      properties:
+        merkle_root:
+          description: Merkle root of the Cardano database snapshot
+          type: string
+          format: bytes
+        beacon:
+          $ref: "#/components/schemas/CardanoDbBeacon"
+        certificate_hash:
+          description: Hash of the associated certificate
+          type: string
+          format: bytes
+        total_db_size_uncompressed:
+          description: Size of the uncompressed Cardano database files in Bytes
+          type: integer
+          format: int64
+        created_at:
+          description: Date and time at which the snapshot was created
+          type: string
+          format: date-time
+        locations:
+          $ref: "#/components/schemas/CardanoDatabaseArtifactsLocationsMessagePart"
+        compression_algorithm:
+          description: Compression algorithm of the Cardano database artifacts
+          type: string
+        cardano_node_version:
+          description: Version of the Cardano node which is used to create snapshot archives.
+          type: string
+      examples:
+        {
+          "merkle_root": "c8224920b9f5ad7377594eb8a15f34f08eb3103cc5241d57cafc5638403ec7c6",
+          "beacon":
+            {
+              "network": "preview",
+              "epoch": 123,
+              "immutable_file_number": 2345
+            },
+          "certificate_hash": "f6c01b373bafc4e039844071d5da3ace4a9c0745b9e9560e3e2af01823e9abfb",
+          "total_db_size_uncompressed": 800796318,
+          "locations":
+            {
+              "digests":
+                [
+                  {
+                    "type": "cloud_storage",
+                    "uri": "https://mithril-cdn-us.iohk.io/snapshot/digests-6367ee65d0d1272e6e70736a1ea2cae34015874517f6328364f6b73930966732.txt"
+                  }
+                ],
+              "immutables":
+                [
+                  {
+                    "type": "cloud_storage",
+                    "uri": "https://mithril-cdn-us.iohk.io/snapshot/immutables-6367ee65d0d1272e6e70736a1ea2cae34015874517f6328364f6b73930966732-{immutable-file_number}.tar.zst"
```

Comment:
Do we need the digest ? Or it is behind a pattern like the file number ?

Reply:
You're right, it's not needed: I removed it